ClawClone/Assets/Scripts/Player.cs

```diff
@@ -143,6 +143,23 @@ protected virtual void OnTriggerEnter2D(Collider2D other)
             checkpoint.Visit();
             _respawnPoint = checkpoint.transform;
         }
+        else if (other.gameObject.layer == LayerMask.NameToLayer("Deadly"))
+        {
+            InvokeRepeating("DeadlyHurt", 0, 1);
```

Comment:
Imo it would be nice to extract function name when we need:
so lets add something like this:
    `
private static string GetFunctionName(Action method)
    {
        return method.Method.Name;
    }
`
and use it when we need. Example:
CancelInvoke(GetFunctionName(DeadlyHurt))

It would be highlited with ctrlf or etc, string will not.